src/components/splashScreen/styles.ts

```diff
@@ -12,14 +13,14 @@ export const splashStyles = createThemedStyles(
         ...layout.flex_1,
         ...layout.justifyCenter,
         ...layout.itemsCenter,
-        backgroundColor: colors.fixedblue700,
+        backgroundColor: colors.blue700,
```

Comment:
what's the diff b/w `fixedblue` and `blue`?

Reply:
fixedBlue remains blue in both the themes(light and dark) but blue changes according to theme